sonic-chassisd/scripts/chassisshow

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+"""
+    chassisshow
+    Chassis information update tool for SONiC
+    This tool runs one time at the launch of the platform monitor in order to populate STATE_DB with chassis information such as model, serial number, and revision.
+"""
+
+try:
+    import os
+    import sys
+
+    from sonic_py_common import daemon_base, logger
+
+    # If unit testing is occurring, mock swsscommon and module_base
+    if os.getenv("CHASSISSHOW_UNIT_TESTING") == "1":
+        from tests import mock_swsscommon as swsscommon
+    else:
+        from swsscommon import swsscommon
+except ImportError as e:
+    raise ImportError(str(e) + " - required module not found")
+
+#
+# Constants
+#
+
+SYSLOG_IDENTIFIER = "chassisshow"
```

Comment:
It looks like the script is to update chassis info, the name "chassisshow" is a litter bit strange to me.

Reply:
What do you suggest? I lack a strong opinion on this but I do have difficulty coming up with a satisfactory name for myself.

We could go with `chassis_update` but that makes it sound like the chassis is being modified in some way.

We could also go with `chassis_info` but that almost implies that if one were to execute it then it would spit out the chassis info (I understand that this is also a problem with the current name)

We could go with `chassis_db_update` but that feels too long (although if you disagree I would be happy with this)

We could go with `chassis_db` but that feels like it would be some separate database program / software. My argument against this one is kind of thin though so if y'all like it then I'm good with it. 

I am open to any suggestions from someone more familiar with the naming conventions of SONiC. Thank you.

Reply:
How about chassis_db_init? Even though the name is little big it would be suited to what the script does.

Reply:
Good with me.